lib/interface_quda.cpp

```diff
@@ -1381,6 +1406,37 @@ void endQuda(void)
 
   if(momResident) delete momResident;
 
+  // Delete MILC FGI fields
```

Comment:
I know we discussed moving these into `freeUniqueGauge`, etc, offline --- just adding this comment so it can be marked as "resolved" at some point.